src/compiler/lsp/protocol_summary.cc

```diff
@@ -318,6 +318,7 @@ void Writer::print_field(ir::Field* field) {
   print_range(field->range());
 
   this->printf("%s\n", field->is_final() ? "final" : "mutable");
+  this->printf("%s\n", field->is_deprecated() ? "deprecated" : "not deprecated");
```

Comment:
Maybe we should try to avoid having the "non-abstract" and "not deprecated" strings contain the positive variant?

Reply:
done. Now simply sending "-" in both cases for the negative case.